linkml_model/model/schema/meta.yaml

```diff
@@ -1420,6 +1420,39 @@ slots:
       - BasicSubset
       - ObjectOrientedProfile
 
+  array_info:
```

Comment:
are there no minimum properties that need to be specified on an array? eg. having this property without any values is an `Any` shaped array?